plugins/GSdx/Renderers/DX11/GSDevice11.h

```diff
@@ -42,11 +42,14 @@ class GSDevice11 final : public GSDevice
 		GSVector4 VertexOffset;
 		GSVector4 Texture_Scale_Offset;
 
+		GSVector2i DepthMask;
```

Comment:
Why isn't the mask stored in a single unsigned integer?

Reply:
When I initially ported it I only followed what was done on gl but looking at it now you're right.
I don't see any use case where two vectors will be used in the shader.